charts/kubeclarity/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.kubeclarity.prometheus.serviceMonitor.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "kubeclarity.name" . }}
+  {{- with (((.Values.kubeclarity).prometheus).serviceMonitor).annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  labels:
+    {{- include "kubeclarity.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "kubeclarity.labels" . | nindent 4 }}
```

Comment:
I think it should be `nindent 6`, right?

Reply:
you are right. Fixed